tomviz/CentralWidget.ui

```diff
@@ -96,15 +68,15 @@
  </widget>
  <customwidgets>
   <customwidget>
-   <class>pqTabbedMultiViewWidget</class>
+   <class>QVTKWidget</class>
    <extends>QWidget</extends>
-   <header>pqTabbedMultiViewWidget.h</header>
+   <header location="global">qvtkwidget.h</header>
```

Comment:
Designer does some really strange things sometimes.  If you ever edit the MainWindow.ui file, there is a two column table in there that you have to set the number of columns back to 2.  Designer is convinced it should be 1 for some reason.